pandas/tseries/tests/test_tslib.py

```diff
@@ -519,7 +519,11 @@ def test_parsers(self):
                  '2014-06': datetime.datetime(2014, 6, 1),
                  '06-2014': datetime.datetime(2014, 6, 1),
                  '2014-6': datetime.datetime(2014, 6, 1),
-                 '6-2014': datetime.datetime(2014, 6, 1), }
+                 '6-2014': datetime.datetime(2014, 6, 1),
+
+                 '20010101 12': datetime.datetime(2001, 1, 1, 12),
```

Comment:
add some cases for ``:`` appearing once, e.g. ``HH:MMSS`` (which I think is invalid)